_layouts/default.html

```diff
@@ -1,14 +1,21 @@
 ---
 ---
+<!-- 
+Copyright 2019 OpenPrinting
 
-<!doctype html>
-<!--
-  Minimal Mistakes Jekyll Theme 4.13.0 by Michael Rose
-  Copyright 2013-2018 Michael Rose - mademistakes.com | @mmistakes
```

Comment:
Minimal Mistakes have copyright for this code. I think we cannot remove their copyright.

Reply:
It's covered under the MIT license which AFAIK allows us to put our license if we modify the work. I am not sure about this though. @tillkamppeter ?

Reply:
I do not know about these license details. @iramcdonald can you help here?
Probably we can put the Apache license but have to continue the original copyright in it with our copyright (for our changes) added.